src/run.ts

```diff
@@ -147,7 +147,7 @@ export async function runPublish({
   let releasedPackages: Package[] = [];
 
   let publishPackageRegex = skipNpm ? GITHUB_TAG_REGEX : NPM_TAG_REGEX;
-  let publishedSucceed = skipNpm ?? changesetPublishOutput.stdout.includes(
+  let publishedSucceed = skipNpm ? true : changesetPublishOutput.stdout.includes(
```

Comment:
Another alternative would be the one below using `||`, but it's crucial to note that none of these will function correctly if _the default value_ is a `"false"` [string](https://github.com/FuelLabs/changesets-action/pull/7/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R42) (which is truthy) instead of a [boolean](https://github.com/FuelLabs/changesets-action/pull/7/files#diff-69c9100f2a3041c22e8fddf77d13fd7eb1aa42a8a200aa35a81ea589c135e66aR134) `false` (which is false).

Would you happen to know for sure how this default `"false"` value is evaluated? (string vs. boolean)
> cc @helciofranco 

```suggestion
  let publishedSucceed = skipNpm || changesetPublishOutput.stdout.includes(
```

Reply:
@arboleya This is definitely a mistake, it should be `boolean`. 

Can we take the opportunity to also fix that in the `action.yml` file, @nedsalk? 

Other thing that is missing is to run the build and commit it and then it looks good to merge.

Reply:
Unfortunately we didn’t catch, we did some tests with nullable values initially. Then we forget to update things. 

Sorry about that guys. Anything let me know.

Reply:
@arboleya you're right. I suppose a helper function that evaluates `"true"` and `"false"` strings alongside their boolean equivalents would cover that.
```ts
function isTrue(skipNpm) {
  switch(skipNpm) {
    case true:
    case "true":
      return true;
    default:
      return false;
  }
}
```
@helciofranco Yes, please feel free to add whatever you think is needed to round this up.

Reply:
Actually, I just looked at it a bit more and [here](https://github.com/FuelLabs/changesets-action/blob/2750e4dcf869d4869a3dd44e1ebd9ffa4f66aa76/src/index.ts#L112) the `core.getBooleanInput` utility is being used so this shouldn't be a problem?

Reply:
@nedsalk yep, we need to fix the default value in the `action.yml` file in order to use the `||` operator.

Reply:
https://github.com/FuelLabs/changesets-action/pull/9/commits/a8cd85b26a861d3460ed7b17c1e286a69ee9a0f4